ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js

```diff
@@ -107,80 +106,71 @@ const SnapAuthorship = ({ snapId, className, expanded = false, snap }) => {
+      <Box flexDirection={FLEX_DIRECTION.COLUMN} width={BLOCK_SIZES.FULL}>
+        <Box
+          flexDirection={FLEX_DIRECTION.ROW}
+          justifyContent={JustifyContent.spaceBetween}
+          paddingLeft={4}
+          paddingTop={4}
+          paddingBottom={4}
+          borderColor={BorderColor.borderDefault}
+          width={BLOCK_SIZES.FULL}
+          style={{
+            borderLeft: BorderStyle.none,
+            borderRight: BorderStyle.none,
+          }}
+        >
+          <Text variant={TextVariant.bodyMdBold}>{t('enableSnap')}</Text>
+          <Box style={{ maxWidth: '52px' }}>
+            <Tooltip interactive position="left" html={t('snapsToggle')}>
+              <ToggleButton value={snap?.enabled} onToggle={onToggle} />
+            </Tooltip>
           </Box>
-          <Box
-            flexDirection={FLEX_DIRECTION.COLUMN}
-            padding={4}
-            width={BLOCK_SIZES.FULL}
-          >
-            {installOrigin && installInfo && (
-              <Box
-                flexDirection={FLEX_DIRECTION.ROW}
-                justifyContent={JustifyContent.spaceBetween}
-                width={BLOCK_SIZES.FULL}
-              >
-                <Text variant={TextVariant.bodyMdBold}>
-                  {t('installOrigin')}
-                </Text>
-                <Box
-                  flexDirection={FLEX_DIRECTION.COLUMN}
-                  alignItems={AlignItems.flexEnd}
-                >
-                  <ButtonLink href={installOrigin.origin} target="_blank">
-                    {installOrigin.host}
-                  </ButtonLink>
-                  <Text color={Color.textMuted}>
-                    {t('installedOn', [
-                      formatDate(installInfo.date, 'dd MMM yyyy'),
-                    ])}
-                  </Text>
-                </Box>
-              </Box>
-            )}
+        </Box>
+        <Box
+          flexDirection={FLEX_DIRECTION.COLUMN}
+          padding={4}
+          width={BLOCK_SIZES.FULL}
+        >
+          {installOrigin && installInfo && (
             <Box
               flexDirection={FLEX_DIRECTION.ROW}
               justifyContent={JustifyContent.spaceBetween}
-              alignItems={AlignItems.center}
-              marginTop={4}
+              width={BLOCK_SIZES.FULL}
             >
-              <Text variant={TextVariant.bodyMdBold}>{t('version')}</Text>
-              <SnapVersion version={snap?.version} url={url} />
+              <Text variant={TextVariant.bodyMdBold}>{t('installOrigin')}</Text>
+              <Box
+                flexDirection={FLEX_DIRECTION.COLUMN}
+                alignItems={AlignItems.flexEnd}
+              >
+                <ButtonLink href={installOrigin.origin} target="_blank">
+                  {installOrigin.host}
+                </ButtonLink>
+                <Text color={Color.textMuted}>
+                  {t('installedOn', [
+                    formatDate(installInfo.date, 'dd MMM yyyy'),
```

Comment:
It doesn't seem to me that we have any existing pattern for this

Reply:
Then we should come up with one, because we can't just assume it's `dd-mm-yyyy` in my opinion. For example:

```ts
date.toLocaleDateString(undefined /* or the user selected locale? */, {
  year: 'numeric',
  month: '2-digit',
  day: '2-digit',
})
```

Reply:
Or just use the [proper format](https://xkcd.com/1179/).

Reply:
Heh, I agree. I just think its out of scope for this PR. Feel free to create a ticket for it!